utils/src/main/java/com/hazelcast/simulator/utils/FileUtils.java

```diff
@@ -426,10 +342,10 @@ public static File getFile(OptionSpec<String> spec, OptionSet options, String de
         return file;
     }
 
-    public static String getFileAsTextFromWorkingDirOrSimulatorHome(String fileName, String desc) {
+    public static String getFileAsTextFromWorkingDirOrBaseDir(File baseDir, String fileName, String desc) {
```

Comment:
The method name may be a bit uglier now, but I had to get rid of the `Coordinator` dependency :(